src/router/handlers/user.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * @swagger
+ * /user/:
+ *  get:
+ *    description: Use to request a user
+ *    parameters:
+ *      - name: ha-api-token
+ *        in: header
+ *        required: true
+ *        schema:
+ *          type: string
+ *          format: uuid
+ *    responses:
+ *      '200':
+ *        description: Success
+ *        schema:
+ *          type: object
+ *          properties:
+ *            user:
+ *              type: object
+ */
+let getUser = async (req, res) => {
+    
+};
+
+/**
+ * @swagger
+ * /user/:
```

Comment:
For updating a user, it will need to define the user id somewhere: `/user/:uid/:`

Reply:
You are right!